Dockerfile

```diff
@@ -82,6 +82,7 @@ RUN set -ex; \
     ; \
     \
     npm config set --global python python2.7; \
+    npm install -g bower; \
```

Comment:
I disagree 👎 
If we have legacy dependency system, install it into a specific `Dockerfile` of the project.

Reply:
For me `bower` is a package manager, can be part of the docker image. But if we don't use it at all, yes we can put it in the local `Dockerfile`